layouts/shortcodes/button.html

```diff
@@ -0,0 +1 @@
+<a class="button" href="{{.Get 0}}" role="button">{{.Get 1}}</a>
```

Comment:
```suggestion
{{- if .IsNamedParams -}}
<a class="button" href="{{ .Get `link` }}" role="button">{{ .Get `text` }}</a>
{{- else -}}
<a class="button" href="{{ .Get 0 }}" role="button">{{ .Get 1 }}</a>
{{- end -}}
```

Dokładnie tak :) Dzięki za obczajenie tego, nie myślałem że to takie proste będzie. Dodaję jeszcze przy okazji obsługę parametrów nazwanych.